test/langtools/tools/javac/8074306/TestSyntheticNullChecks.java

```diff
@@ -27,7 +27,7 @@
  * @summary NULLCHK is emitted as Object.getClass
  * @compile -source 7 -target 7 TestSyntheticNullChecks.java
  * @run main TestSyntheticNullChecks 7
- * @clean TestSyntheticNullChecks*
+ * @clean TestSyntheticNullChecks
```

Comment:
This test class has an inner class.  If the original intent was to try to remove all class files, the change is not equivalent, and will not do that; it will clean the main class, but not the nested class.

You may want to use `@clean *` instead, which will clean (remove) all class files in the default package.

Reply:
Right, fixed.